.github/workflows/check-new-links.yml

```diff
@@ -0,0 +1,40 @@
+name: Check new links
+
+on:
+  pull_request:
+
+jobs:
+  check_urls:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Find changed lines
+        id: changed_lines
+        uses: hestonhoffman/changed-lines@v1
+        with:
+          file_filter: .txt
+
+      - name: Find any URLs in new lines and check they are resolvable
```

Comment:
I have seen this being used before https://github.com/marketplace/actions/urlchecker-action
Not sure would work here to make things simpler?

Reply:
> I have seen this being used before https://github.com/marketplace/actions/urlchecker-action Not sure would work here to make things simpler?

I only want to check new links - not all links on the page.
And I want to catch invalid but URL-alike links as well.